symplyphysics/laws/electricity/circuits/specific_resistance_of_coaxial_waveguide.py

```diff
@@ -0,0 +1,56 @@
+from sympy import (Eq, solve, pi, sqrt,)
+from symplyphysics import (units, Quantity, Symbol, print_expression, validate_input,
+    validate_output, dimensionless,)
+from sympy.physics.units import magnetic_constant
+
+
+# Description
+## A coaxial waveguide is an electrical cable consisting of a central conductor and a shield arranged coaxially and separated
+## by an insulating material or an air gap. It is used to transmit radio frequency electrical signals.
+## The specific resistance of a coaxial waveguide depends on the radius of the outer conductor and the radius of the inner conductor,
+## as well as on the relative permeability of the insulator material, frequency of signal and specific conductivity of conductor.
+
+## Law is: R = (1 / (2 * pi)) * sqrt(w * mu0 * mur / (2 * sig)) * (1 / a - 1 / b), where
+## R - specific resistance of coaxial waveguide,
+## w - frequency of signal,
+## sig - specific conductivity of conductor,
+## mu0 - magnetic constant,
+## mur - relative permeability of the insulator material,
+## b - radius of the outer conductor,
+## a - radius of the inner conductor.
+
+specific_resistance = Symbol("specific_resistance", units.impedance / units.length)
+
+relative_permeability = Symbol("relative_permeability", dimensionless)
+frequency = Symbol("frequency", 1 / units.time)
```

Comment:
Use `units.frequency`
If it is angular frequency, use `angle_type / units.time`